raiden-dapp/src/components/Tokens.vue

```diff
@@ -48,7 +48,7 @@
               <v-layout justify-center row>
                 <v-btn
                   :id="`pay-${index}`"
-                  :to="`/transfer/${token.address}`"
+                  :to="`/send-transaction/${token.address}`"
```

Comment:
I'd say it shouldn't be called `send-transaction`, as transaction remembers on-chain transactions (txs), while payments/transfers on Raiden are fully off-chain and aren't related to actual transactions. Raiden Python calls these just payments, in the sence that, in the future, a single payment could be split in multiple transfers. We should stick with this terminology here.

Reply:
Yes, it makes more sense to follow the same semantics and terminology as the python documentation. We will rename it to payment then for consistency.